frontend/src/components/pages/CreateReview/AddStringInput.tsx

```diff
@@ -0,0 +1,60 @@
+import { FormControl, FormLabel, Image } from "@chakra-ui/react";
+import React from "react";
+
+import InputField from "./InputField";
+
+type AddStringInputProps = {
+  id: string;
+  label: string;
+  name: string;
+  placeholder?: string;
+  required: boolean;
+  maxWidth?: string;
+  bookCover?: boolean;
+  inputField: string;
+  setInputField: (s: string) => void;
+};
+
+const AddStringInput = ({
+  id,
+  label,
+  name,
+  placeholder,
+  required,
+  maxWidth,
+  bookCover,
+  inputField = "",
+  setInputField,
+}: AddStringInputProps): React.ReactElement => {
+  // const [inputFields, setInputFields] = useState<string[]>([""]);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputField(e.target.value);
+  };
+
+  return (
+    <FormControl id={id} isRequired={required} width={maxWidth || "100%"}>
+      <FormLabel mb={2}>{label}</FormLabel>
+      <InputField
+        id={0}
+        key={0}
+        name={name}
+        placeholder={placeholder || "Text Here"}
+        required={required}
+        handleDelete={() => {}}
+        value={inputField}
+        handleInputChange={handleInputChange}
+      />
+      {bookCover && inputField.length > 0 ? (
```

Comment:
Why do we need this in the AddStringInput component? I think I missed how it's being used

Reply:
Oh yea no we don't need this, good catch!